codespell_lib/data/dictionary.txt

```diff
@@ -7599,6 +7679,7 @@ evyrthing->everything
 ewhwer->where
 exaclty->exactly
 exacly->exactly
+exacta->exacte
```

Comment:
`exacta` is a [valid word](https://www.usracing.com/horse-betting/exacta) and `exacte` isn't.